src/ViewHelpers/ConstantViewHelper.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file belongs to the package "TYPO3 Fluid".
+ * See LICENSE.txt that was shipped with this package.
+ */
+
+namespace TYPO3Fluid\Fluid\ViewHelpers;
+
+use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
+use TYPO3Fluid\Fluid\Core\ViewHelper\AbstractViewHelper;
+use TYPO3Fluid\Fluid\Core\ViewHelper\Traits\CompileWithRenderStatic;
+
+/**
+ * Wrapper for PHPs :php:`constant` function.
+ * See https://www.php.net/manual/function.constant.php.
+ *
+ * Examples
+ * ========
+ *
+ * Get built-in PHP constant
+ * -------------------------
+ *
+ * ::
+ *
+ *    {f:constant(value: 'PHP_INT_MAX')}
+ *
+ * Output::
+ *
+ *    9223372036854775807
+ *    (Depending on CPU architecture).
+ *
+ * Get class constant
+ * ------------------
+ *
+ * ::
+ *
+ *    {f:constant(value: '\Vendor\Package\Class::CONSTANT')}
+ *
+ * Get enum value
```

Comment:
```suggestion
 * Get enum instance
```

Reply:
Actually `value` is the correct term for items of enumerations. There are no instances, even though enums in PHP may behave like such.

Reply:
An instance represents a value but it's IMO still an instance. That term is also used in the PHP documentation.

https://www.php.net/manual/en/language.enumerations.overview.php
> ... The Enum itself is a class, and its possible cases are all single-instance objects of that class. ...

https://www.php.net/manual/en/language.enumerations.object-differences.php
> Cloning an Enum case is not supported, as cases must be singleton instances.

Reply:
I think, in the context of Fluid, `value` covers >95% of use cases, since methods can't be called anyways from Fluid (apart from get/is/has).